image/templates/helm/shared/templates/_openshift.tpl

```diff
@@ -45,3 +45,41 @@
 {{ end }}
 
 {{ end }}
+
+{{/*
+    srox.initOpenshiftMonitoring $
+
+    This function enables the Openshift Monitoring variables and flags.
+*/}}
+{{ define "srox.initOpenshiftMonitoring" }}
+
+{{ $ := index . 0 }}
+
+{{ if $._rox.enableOpenShiftMonitoring }}
+  {{ include "srox.warn" (list . "enableOpenShiftMonitoring option was replaced with monitoring.openshift.enabled") }}
+  {{ $_ := set $._rox "monitoring" (dict "openshift" (dict "enabled" true)) }}
+{{ end }}
+
+{{/* Default `monitoring.openshift.enabled = true` unless `env.openshift != 4`. */}}
+{{ if kindIs "invalid" $._rox.monitoring.openshift.enabled }}
+  {{ $_ := set $._rox "monitoring" (dict "openshift" (dict "enabled" (eq $._rox.env.openshift 4))) }}
+{{ end }}
+
+{{ if and $._rox.monitoring.openshift.enabled (ne $._rox.env.openshift 4) }}
+  {{ include "srox.warn" (list . "'monitoring.openshift.enabled' is set to true, but the chart is not being deployed in an OpenShift 4 cluster. Proceeding with 'monitoring.openshift.enabled=false'.") }}
+  {{ $_ := set $._rox "monitoring" (dict "openshift" (dict "enabled" false)) }}
+{{ end }}
+
+{{/*
+  Detect if the secured-cluster-services chart is deployed in the same namespace
```

Comment:
Keep in mind that Central may be installed multiple times and into different namespaces. For example in the cloud service there are many Central charts installed into `rhacs-{tenant-id}` namespaces on the same cluster.